samples/sagemaker_huggingface_model/package-lock.json

```
@@ -5724,8 +5724,7 @@
     "cdk-nag": {
       "version": "2.28.17",
       "resolved": "https://registry.npmjs.org/cdk-nag/-/cdk-nag-2.28.17.tgz",
-      "integrity": "sha512-dmM8mvegWEKfEffWWZk6hY/33PdnOuWahCC+Bs/u4nYSRLt1AuSr4znbfyib3WyAgfc6E18Aem0Yodh9h6iuJA==",
-      "requires": {}
+      "integrity": "sha512-dmM8mvegWEKfEffWWZk6hY/33PdnOuWahCC+Bs/u4nYSRLt1AuSr4znbfyib3WyAgfc6E18Aem0Yodh9h6iuJA=="
```

All the other lines end with comma, `,`.

```suggestion
      "integrity": "sha512-dmM8mvegWEKfEffWWZk6hY/33PdnOuWahCC+Bs/u4nYSRLt1AuSr4znbfyib3WyAgfc6E18Aem0Yodh9h6iuJA==",
```

---

suggestion accepted and commited.